Modules/Options/TrackerTab/QuestieOptionsTracker.lua

```diff
@@ -73,11 +76,14 @@ function QuestieOptions.tabs.tracker:Initialize()
                 order = 1.3,
                 width = 1.0,
                 name = function() return l10n('Min Complete Quests'); end,
-                desc = function() return l10n('When this is checked, completed quests will automatically minimize.'); end,
+                desc = function() return l10n('When this is checked, completed quests will automatically minimize.\n\nNOTE: If this option is currently enabled, disabling this option will maximize ALL minimized quests currently in your Questie Tracker.'); end,
```

Comment:
When chaning the content of a `l10n` call you also need to change [the translation](https://github.com/Questie/Questie/tree/master/Localization/Translations)

Reply:
Copy that... I'll do my best. :) I forgot about updating all the translations.